### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /notebook-ci-testing/README.html

### ISSUE THREAD

**Issue by dulude** [dulude/notebook-ci-testing#1]
## What
  An accessibility scan flagged the element `<a class="reference external" href="https://github.com/spacetelescope/jwst-pipeline-notebooks/blob/main/notebooks/MIRI/Coronagraphy/JWPipeNB-MIRI-Coron.ipynb">JWPipeNB-MIRI-Coron.ipynb</a>` on https://spacetelescope.github.io/notebook-ci-testing/README.html because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

  
[View screenshot](https://github.com/dulude/notebook-ci-testing/blob/gh-cache/.screenshots/ff1ff66d-c70a-43cb-8032-e0ee5d99d87a.png)

  To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.
  
Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 4.34 (foreground color: #176de8, background color: #f3f4f5, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

  ## Acceptance Criteria
  - [ ] The specific axe violation reported in this issue is no longer reproducible.
  - [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
  - [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
  - [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
